exasol/analytics/query_handler/query/result/udf_query_result.py

```diff
@@ -92,7 +90,8 @@ def _compute_columns(self, exa) -> List[Column]:
             column.name: column.sql_type for column in exa.meta.input_columns
         }
         columns = [
-            Column(ColumnName(key), ColumnType(column_dict[value]))
+            Column.from_sql_type(key, column_dict[value])
```

Comment:
Actually, this is location is unclear to me. 

1) The former implementation instantiated `ColumnType` with argument name set to `VARCHAR(20)`, for example.
AFAIK, column type names should not contain numeric arguments, as type Column also contained an attribute `size`, for example.

2) Additionally, these arguments are also not passed to `exa.meta.input_columns` in UDFs. 

3) Some tests used column types like `INTEGER` and `FLOAT`, which actually are mapped to `DECIMAL` and `DOUBLE` and hence cannot be reproduced in reality. 

All in all the former implementation and the related tests do not seem 100% plausible to me.

Reply:
I think, that happened out of lack of time, your implementation looks more correct

Reply:
OK, then let's go on with the current implementation and address potential issues as soon as we encounter these.